.github/workflows/dockerhubpublish.yml

```diff
@@ -7,11 +7,9 @@ on:
 
   workflow_dispatch:
     inputs:
-      is_workflow_dispatch:
-        description: keep as 1
-        type: number
-        required: true
-        default: 1
+      version:
+        description: 'Version to bump to'
+        required: false
```

Comment:
afaict this is never read/used, still uses `RELEASE_VERSION=$(git describe --tags --abbrev=0)`

Reply:
nice!

yeah not sure how to do the null check but conceptually yeah, coalesce as part of step `get most recent tag`

Reply:
Yea something like that also with an else for the echo $release_version = $(git describe tags --abbrev=0) >> github_env part